pkg/common/cns-lib/volume/listview.go

```diff
@@ -3,12 +3,15 @@ package volume
 import (
 	"context"
 	"errors"
+	"fmt"
 	"time"
 
 	"github.com/vmware/govmomi"
 	"github.com/vmware/govmomi/property"
 	"github.com/vmware/govmomi/session"
 	"github.com/vmware/govmomi/view"
+	"github.com/vmware/govmomi/vim25/methods"
```

Comment:
do we need to update go.mod & go.sum to update the govmomi release since the ModifyListView function with fix has been used below ?

Reply:
Please refer to this comment to see how we are using `ModifyListView()` https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2592#discussion_r1356325975. 

We are simply using a different method that is already available in the library. Once the govmomi fix gets into a release, we can upgrade to it and can revert back to calling Add() method directly.

Reply:
Based on offline discussion, this is something which is not needed immediately and we are ok to wait until a govmomi release